README.md

```diff
@@ -6,34 +6,35 @@
 <img src="assets/NightDriverLogo-small.png" width="400" />
 
 <!-- markdownlint-disable MD036 /no-emphasis-as-heading -->
+

```

Comment:
What may actually be best here is to change `markdownlint-disable` to `markdownlint-disable-next-line` and then keep it together with the line it is actually meant to address. That does mean the empty line you added will need to be removed again.

Reply:
Can't really comment on this since I don't use markdownlint, but sure, makes sense.